File: Web/RMA-Member-Portal/memberportal-app/src/app/shared/models/linked-user-member.ts
import { UserCompanyMapStatusEnum } from "../enums/user-company-map-status-enum";

export class LinkedUserMember {
  rolePlayerId: number;
  memberName: string;
  finPayeNumber: string;
  userCompanyMapStatus: UserCompanyMapStatusEnum;
  roleId: number;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/LinkedUserMember.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class LinkedUserMember
    {
        public int RolePlayerId { get; set; }
        public string MemberName { get; set; }
        public string FinPayeNumber { get; set; }
        public UserCompanyMapStatusEnum UserCompanyMapStatus { get; set; }
        public int RoleId { get; set; }
    }
}